easybuild/easyconfigs/c/Caffe2/Caffe2-0.8.1-CrayGNU-17.08-cuda-8.0-python3.eb

```diff
@@ -0,0 +1,82 @@
+# @author: gppezzi, mschoengens
+
+easyblock = 'CMakeMake'
+
+name = 'Caffe2'
+version = '0.8.1'
```

Comment:
This is misleading because you are not actually installing version `0.8.1` so please use something else as version (and update the file name). For example, use 7 digits from the commit hash or use a date if you prefer.